docs/data/apis/README.mdx

```diff
@@ -0,0 +1,36 @@
+---
+title: APIs Overview
+sidebar_position: 0
+---
+
+Learn about the services that provide access to real-time Stellar network data
+
+| Features                | RPC | Horizon |
+| ----------------------- | --- | ------- |
+| Real-time Data          | ✅  | ✅      |
+| Historical Data         | ❌  | ❌\*    |
+| Smart Contracts         | ✅  | ❌      |
+| Transaction Simulation  | ✅  | ❌      |
+| Curated and Parsed Data | ❌  | ✅      |
+
+\*_Please note that Horizon can provide full historical data but is not the recommended tool for full historical data access. Please use [Hubble](../analytics/hubble/README.mdx) or [Galexie](../indexing/build-your-own/galexie/README.mdx) instead._
+
+## [RPC](./rpc/README.mdx)
+
+The RPC provides real-time access to the current state of the Stellar network, including account balances, smart contract states, and recent transaction queries (within a seven-day retention window), while also allowing transaction submission. It is designed to be simple, minimal, and scalable, making it ideal for applications and wallets that require live data availability.
+
+:::note
+
+RPC is the recommened API service for all users to use.
+
+:::
+
+## [Horizon](./horizon/README.mdx)
+
+Horizon is an API for accessing and interacting with Stellar network data, storing current, historical, and derived state in a single database but without smart contract data. It provides real-time data for transactional use, making it more resource-intensive to operate, and can be run independently or accessed through public infrastructure providers. Depending on the [Ecosystem Horizon Provider](./horizon/horizon-providers.mdx), Horizon instances may contain 1 year of data or full historical data for the Stellar network.
```

Comment:
this is a little misleading. Horizon provides _limited_ smart contract support, ie, i think you can track movement of SAC tokens through payments and asset_stats endpoints

@mollykarcher do you think it's worth mentioning?

Reply:
Eh, I think this is fine to leave as-is. Horizon only supports SAC because it _wants_ to support Classic Asset movements, so if it didn't support the SAC it would lose that information.